kafka-transport-binding.md

```diff
@@ -102,10 +100,10 @@ here.
 
 The receiver of the event can distinguish between the two content modes by
 inspecting the `ce_datacontenttype` [Header][Kafka-Message-Header] of the 
-Kafka message. If the value is prefixed with the CloudEvents media type
-`application/cloudevents`, indicating the use of a known
-[event format](#14-event-formats), the receiver uses *structured* mode, otherwise
-it defaults to *binary* mode.
+Kafka message. If no value is given, or if the value is prefixed with the
```

Comment:
If I'm correct, and it should be `content-type` above, then I think "no value" should default to binary not structured, and then the original text would make more sense.

Reply:
How will this work with Apache Kafka < 0.11 then though (also commented about this on your new PR)? Headers cannot be used there, so all the other `ce_...` attributes couldn't be given. Am I missing something?

Reply:
MQTT <= 3.1 >= 4 has a similar issue. The approach there was MUST use JSON format and not support anything else on older MQTT versions.